share/ipld/get.go

```diff
@@ -31,6 +32,9 @@ var NumWorkersLimit = MaxSquareSize * MaxSquareSize / 2 * NumConcurrentSquares
 // concurrently/simultaneously.
 var NumConcurrentSquares = 8
 
+// ErrNotFound is used to signal when a nmt Node could not be found.
+var ErrNotFound = errors.New("nmt node not found")
```

Comment:
Because both errors are exported and called ErrNotFound it may be confusing at some point. I won't block on this because its just an idea, but maybe rename to something else like ErrNodeNotFound

Reply:
I like it, thank you!